test/test_nestedtensor.py

```diff
@@ -490,15 +490,26 @@ def test_fill_(self):
             t.fill_(11.)
             self.assertEqual(nt_ub, t)
 
-    def test_ones_like(self):
+    def test_zero_(self):
         ntensors = 4
         nt = random_nt(torch.device('cpu'), torch.float32, ntensors, (4, 4))
-        ones_nt = torch.ones_like(nt)
-
-        for nt_ub in ones_nt.unbind():
-            t = torch.ones_like(nt_ub)
+        nt.zero_()
+        for nt_ub in nt.unbind():
+            t = torch.empty_like(nt_ub)
+            t.fill_(0.)
             self.assertEqual(nt_ub, t)
 
+    @parametrize("func", [torch.ones_like, torch.zeros_like],
```

Comment:
If this introduces spaces or weird symbols the internal test runner might not like it. You can change the display name by using explicit names. See https://github.com/pytorch/pytorch/blob/master/test/test_sparse_csr.py#L207

Reply:
on the next line, I use `name_fn=lambda f: f.__name__` to ensure readable test names :)